src/pages/Home.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import CountdownTimer from "../components/CountdownTimer";
+
+const Home = () => (
+  <div className="Home" data-test="Home-page">
+    <h1>SnapIT Hacktoberfest Starts In:</h1>
+    <CountdownTimer data-test="countdown-display" />
+  </div>
+);
+
+export default Home;
```

Comment:
:wrench: This file should be removed.

Reply:
Addressed:  f639309aca79206b6c8cc424c70be42e6adfb37b